mk/android.mk

```diff
@@ -1,7 +1,13 @@
-#CC=arm-linux-androideabi-gcc
+ifeq (${PATH},"/usr/arm-linux-androideabi/bin/${ARCH}-linux-androideabi-gcc")
```

Comment:
That is the path to gcc as it is inside dockcross :-S How would you specify this otherwise?